server/network/webhooks.py

```diff
@@ -89,6 +89,37 @@ def modcall(self, char, ipid, area, reason=None):
             description=description,
         )
 
+    def needcall(self, client=None, reason=None):
+        is_enabled = self.server.config["need_webhook"]["enabled"]
+        username = self.server.config["need_webhook"]["username"]
+        avatar_url = self.server.config["need_webhook"]["avatar_url"]
+        color = self.server.config["need_webhook"]["color"]
+        title = self.server.config["need_webhook"]["title"]
+        url = self.server.config["need_webhook"]["url"]
+        pingoption = self.server.config["need_webhook"]["pingoption"]
```

Comment:
Please first check if need_webhook is a thing, otherwise this will error for servers that didn't set it up yet